reinforcement_learning/rl_portfolio_management_coach_customEnv/rl_portfolio_management_coach_customEnv.ipynb

```diff
@@ -557,6 +557,7 @@
     "                      entry_point=\"evaluate-coach.py\",\n",
```

Comment:
<p>Looks like this notebook is really old (coach 0.11.0) and probably should be one of the first batch to convert to Ray.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>